share/completions/rustup.fish

```diff
@@ -296,3 +296,6 @@ end
 function __rustup_doc_path
     rustup doc --path | string replace --regex '/[^/]*$' ''
 end
+
+set -e __rustup_channels
```

Comment:
These should just be local-exported (`-lx`). That way functions that need them get a copy.